hudi-common/src/main/java/org/apache/hudi/common/fs/HoodieWrapperFileSystem.java

```diff
@@ -79,8 +82,16 @@ public static void setMetricsRegistry(Registry registry, Registry registryMeta)
     METRICS_REGISTRY_META = registryMeta;
   }
 
+  // Minimum buffer size of data files and log files which are generally large in size
+  public static final int MIN_DATA_FILE_IO_BUFFER_SIZE = 16 * 1024 * 1024; // 16 MB
```

Comment:
rename: `MIN_HOODIE_FILE_IO_BUFFER_SIZE` :)

Reply:
I think this is small enough to not a large overhead even if S3/GCS are already buffering.

Reply:
Renamed